src/Desktop.Analyzers/VisualBasic/Helps/SyntaxNodeHelper.vb

```diff
@@ -144,7 +144,6 @@ Namespace Desktop.Analyzers.Common
             Return Nothing
         End Function
 
```

Comment:
Please rename the directory name to `Helpers` to be consistent with other projects.